test/test-find-python.js

```diff
@@ -334,6 +337,6 @@ test('find python - no python, no python launcher, bad guess', function (t) {
   f.checkPython()
 
   function done(err, python) {
-    t.ok(/Can't find Python executable/.test(err))
+    t.ok(/For more information consult the documentation/.test(err))
```

Comment:
This message does not help the user to understand what when wrong and what documentation topics to be focusing on.